app/serializers/component_serializer/card_component_serializer.rb

```diff
@@ -4,6 +4,9 @@ class CardComponentSerializer < ComponentSerializer::BaseComponentSerializer
     #
     # @param [String] name name of the card.
     # @param [Hash] data data of the card.
+    #
+    # @example Initialising a card component
+    #   ComponentSerializer::CardComponentSerializer.new('card__generic', {card_type: 'small', heading: ComponentSerializer::HeadingComponentSerializer.new('See heading for arguments'), paragraph: ComponentSerializer::ParagraphComponentSerializer.new('See paragraph for arguments'), figure: ComponentSerializer::FigureComponentSerializer.new('See figure for arguments'), link_button: ComponentSerializer::LinkButtonComponentSerializer.new('See link button for arguments'), count: ComponentSerializer::CountComponentSerializer.new('See count for arguments')}).to_h
```

Comment:
I have an issue with this because it won't work? I'm happy to be convinced otherwise?

Reply:
I have combined the this comment below. To make the examples both functional and informative I have added variables to the examples. The names of the variables explain what each one is and the content works as an example. It also reduced the length of some of the examples(though not all of them).